src/main/java/org/jenkinsci/plugins/casc/integrations/GroupPermissionDefinition.java

```diff
@@ -0,0 +1,47 @@
+package org.jenkinsci.plugins.casc.integrations;
+
+import hudson.security.Permission;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+import java.util.*;
+
+/**
+ * Created by mads on 2/7/18.
+ */
+public class GroupPermissionDefinition {
+
+    private String name;
+    private Collection<String> permissions = new ArrayList<>();
+
+    @DataBoundConstructor
+    public GroupPermissionDefinition(String name, Collection<String> permissions) {
+        this.name = name;
+        this.permissions = permissions;
```

Comment:
Create a new collection. You can easily get a non-modifyable collection from API

Reply:
done